src/platform/datapath_epoll.c

```diff
@@ -18,6 +18,7 @@
 #include <linux/filter.h>
 #include <linux/in6.h>
 #include <netinet/udp.h>
+#include <stdatomic.h>
```

Comment:
nit: this is already included from platform_internal.h so it's unnecessary here too.